python/cudf/cudf/tests/text/test_text_methods.py

```diff
@@ -885,65 +885,45 @@ def test_is_vowel_consonant():
 def test_minhash():
```

Comment:
?
```suggestion
def test_minhash_permuted():
```

Reply:
I did the pylibcudf one but missed this one.